app/src/components/shared/EventListItem.svelte

```diff
@@ -19,18 +19,16 @@
       {event.title}
     </h2>
     <div class="flex flex-row items-center gap-2">
-      <Badge rounded class="h-6 border border-black bg-transparent dark:bg-zinc-800"
-        >{event.category}</Badge
-      >
       {#if event.openForExternals}
-        <Badge rounded class="h-6 border border-black bg-transparent dark:bg-zinc-800"
-          >For alle</Badge
-        >
+        <Badge rounded class="h-6 border border-black bg-transparent dark:bg-zinc-800">Alle</Badge>
```

Comment:
Hva tenker du? Regrupperte også

Reply:
Regrupperingen funker bra! Jeg følgte bare skissene. Men litt skeptisk til bare "Alle", kan det bli vanskelig å skjønne hva det betyr? Står jo f.eks. "Alle" rett over blant kategoriene, så kan virke som det betyr alle kategorier, Hva med om vi har "For alle" og "Kun interne" i stedet? Så blir det mer likt 😊

Reply:
Enig med deg 👍  ser det blir brukt noe lignende inne på arrangementet også 😃